pkg/plugins/vmbased/metadata_v1.go

```diff
@@ -0,0 +1,346 @@
+// Copyright 2018 The OpenPitrix Authors. All rights reserved.
```

Comment:
The file has already been deleted, have you changed any code in the file？

Reply:
No,it is a wrong commit.I will remove it.